src/components/input/IssueFilterDropdowns.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+    <div class="d-flex ga-4 align-center h-100">
```

Comment:
```suggestion
    <div class="d-flex ga-2 align-center h-100">
```

for consistency with the spacing in line above